src/Windows/Avalonia.Win32/Input/WindowsMouseDevice.cs

```diff
@@ -8,7 +8,7 @@
 
 namespace Avalonia.Win32.Input
 {
-    public class WindowsMouseDevice : MouseDevice
+    class WindowsMouseDevice : MouseDevice
```

Comment:
Any particular reason you made this internal? Not that it would affect anything, just interested to know.